src/convertToObject.js

```diff
@@ -3,16 +3,44 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesArray = sourceString.split('\n');
+  const trimmed = [];
+
+  stylesArray.forEach((style) => {
+    trimmed.push(style.trim().replace(';', ''));
```

Comment:
You are replacing only the first occurrence of ';' which might lead to incorrect results if a property value contains ';' within it. Instead of replace, you should use replaceAll.